SPECS/initramfs/initramfs.spec

```diff
@@ -89,7 +89,7 @@ cat > /dev/null \
 if [ -f %{_localstatedir}/lib/rpm-state/initramfs/regenerate ]; then \
     echo "(re)generate initramfs for all kernels," %* >&2 \
     mkinitrd -q \
-    mv /boot/initrd.img-*mshv* /boot/efi/ \
+    [ ! -f /boot/initrd.img-*mshv* ] || mv /boot/initrd.img-*mshv* /boot/efi/ \
```

Comment:
nitpick
Should we have a simple `if [check]; then` instead of this :)
It took me a while to understand what was happening.

Thanks

Reply:
The double negative tripped me up too.

Instead of doing any if conditionals, I suggest doing the following to just suppress the failure exit code from the `mv` if the file didn't exist. Seems to be a fairly common pattern in other install/uninstall RPM scriptlets.
```
mv /boot/initrd.img-*mshv* /boot/efi/ || :
```

Reply:
Thought about this some more, and that'd still result in the error being printed during RPM transactions, which isn't great. Looks like a lot of other specs are using a `>/dev/null 2>&1 || :` pattern in this scenario.

```
mv /boot/initrd.img-*mshv* /boot/efi/ >/dev/null 2>&1 || :
```